engine/src/flutter/lib/ui/semantics/semantics_update_builder_unittests.cc

```diff
@@ -131,5 +131,48 @@ TEST_F(SemanticsUpdateBuilderTest, CanHandleSemanticsRole) {
   DestroyShell(std::move(shell), task_runners);
 }
 
+TEST_F(SemanticsUpdateBuilderTest, CanHandleSemanticsLocale) {
+  auto message_latch = std::make_shared<fml::AutoResetWaitableEvent>();
+
+  auto nativeSemanticsUpdate = [message_latch](Dart_NativeArguments args) {
+    auto handle = Dart_GetNativeArgument(args, 0);
+    intptr_t peer = 0;
+    Dart_Handle result = Dart_GetNativeInstanceField(
+        handle, tonic::DartWrappable::kPeerIndex, &peer);
+    ASSERT_FALSE(Dart_IsError(result));
+    SemanticsUpdate* update = reinterpret_cast<SemanticsUpdate*>(peer);
+    SemanticsNodeUpdates nodes = update->takeNodes();
+    ASSERT_EQ(nodes.size(), (size_t)1);
+    auto node = nodes.find(0)->second;
```

Comment:
nit: worth checking the find result.  if the find fails you'll crash the whole test runner.